include/sys/dmu_zfetch.h

```diff
@@ -36,8 +36,6 @@
 extern "C" {
 #endif
 
-extern uint64_t	zfetch_array_rd_sz;
```

Comment:
Needs to be removed from the `zfs(4)` man page.

Reply:
Sorry, missed that. Done.